src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/FloatingBaseEstimatorIO.h

```diff
@@ -38,11 +39,12 @@ struct InternalState
     Eigen::Vector3d gyroscopeBias; /**< Bias of the gyroscope expressed in the IMU frame */
 
     Eigen::Vector3d imuAngularVelocity; /**< angular velocity of the IMU with respect to the inertial frame expressed in inertial frame, typically unused for strap-down IMU based EKF implementations*/
-    std::map<int, BipedalLocomotion::Contacts::EstimatedContact> supportFrameData; /**< contact measurements */
+    std::map<int, BipedalLocomotion::Contacts::EstimatedContact> supportFrameData; /**< estimated contacts */
+    std::map<int, BipedalLocomotion::Contacts::EstimatedLandmark> landmarkData; /**< estimated landmarks */
```

Comment:
If you don't need the elements to be ordered, I suggest using ``unordered_map`` as it improves the performances with no cost.

Reply:
I had changed it to `map` from `unordered_map`, since I wanted to have a sorted order of indices, so that the sub-blocks in the covariance matrix relevant to multiple landmarks is accessible in a specific order.